src/types/interfaces.ts

```diff
@@ -78,3 +94,122 @@ export interface IStartSessionParams {
   sessionType: SessionTypes;
   jid?: string;
 }
+
+/**
+ * id: sessionId
+ * mute: update the conversation's mute status to match this value
+ */
+export interface ISessionMuteRequest {
+  id: string;
+  mute: boolean;
+}
+
+/**
+ * Most basic params for a call participant that come from the api: /api/v2/conversations/calls/{conversationId}
+ * this is not an exhaustive list, just the ones we currently care about.
+ * NOTE: the `participants` in the /api/v2/conversations/{conversationId} api are slightly different, e.g. no `user` object
+ */
+export interface IConversationParticipant {
+  id: string;
+  address: string;
+  purpose: string;
+  state: string;
+  direction: string;
+  userId?: string;
+  muted: boolean;
+  videoMuted?: boolean;
+  confined: boolean;
+}
+
+export interface IJingleSession extends WildEmitter {
```

Comment:
It is so nice to have typings for the Jingle stuff :)

Reply:
Heh, it's kind of a hack to do it this way, but it's kind of a stop gap for now.